api/apps/restful_apis/aimlapi_api.py

```
@@ -0,0 +1,192 @@
+
+@manager.route("/llm/aimlapi/authorize/start", methods=["POST"])  # noqa: F821
+@login_required
+async def aimlapi_authorize_start():
+    """Create an AIMLAPI device-authorization request and return the consent URL."""
+    partner_id = _partner_id()
+    if not partner_id:
+        return get_data_error_result(message="AIMLAPI partner id is not configured. Set AIMLAPI_PARTNER_ID.")
+
+    try:
+        body = await get_request_json()
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)